rtplibrary/src/main/java/com/pedro/rtplibrary/view/OffScreenGlThread.java

```diff
@@ -49,6 +53,12 @@ public OffScreenGlThread(Context context) {
     this.context = context;
   }
 
+  public OffScreenGlThread(Context context, boolean isFlipStreamHorizontal, boolean isFlipStreamVertical) {
```

Comment:
why is it in constructor?

Reply:
Well, I thought, that if for `OpenGlView` you can set `isStreamFlipHorizontal` and `isStreamFlipVertical` in `xml` attrubutes, the natural way for `OffScreenGlThread`, which is not in xml, would be to set them in constructor.

Reply:
But according to comment https://github.com/marcin-adamczewski/rtmp-rtsp-stream-client-java/pull/2#discussion_r430453251 I'll remove it.